src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -9864,9 +9859,8 @@ Vec3f D_80854814 = { 0.0f, 0.0f, 200.0f };
 f32 D_80854820[] = { 2.0f, 4.0f, 7.0f };
 f32 D_8085482C[] = { 0.5f, 1.0f, 3.0f };
 
-#ifdef NON_MATCHING
-// stack alloc differences
-void Player_UpdateCommon(Player* this, GlobalContext* globalCtx, Input* input) {
+void Player_UpdateCommon(Player* this, GlobalContext* gc, Input* input) {
+    GlobalContext* globalCtx = gc;
```

Comment:
can you name `gc` as `globalCtx2`. we already know why they had a temp for it in some cases so it will be changed later, but globalCtx2 will help us find and replace later

also, do you need to use the arg version for the state frames access? it makes sense if so, since that var is part of the base gamestate. just want to make sure its required

Reply:
yeah `globalCtx2` needs to be used to access gameplayFrames, or else stack gets changed